lib/DDG/Spice/Anime.pm

```diff
@@ -0,0 +1,28 @@
+package DDG::Spice::Anime;
+
+use strict;
+use DDG::Spice;
+
+name "Anime";
+description "Anime information from Hummingbird";
+source "Hummingbird";
+primary_example_queries "naruto anime";
+secondary_example_queries "naruto hummingbird";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/Anime.pm";
+topics "entertainment", "everyday";
+category "entertainment";
+attribution github => ['https://github.com/iambibhas','Bibhas'],
+           twitter => ['https://twitter.com/bibhasdn','Bibhas D'];
+
+spice proxy_cache_valid => "200 30d";
+spice to => 'https://hummingbird.me/api/v1/search/anime?query=$1';
+
+triggers startend => 'anime', 'hummingbird';
```

Comment:
@iambibhas sorry just saw this! I would suggest we use both methods to ensure we don't make redundant API calls, and only show relevant results to users